react-app/webpack/webpack.common.js

```diff
@@ -0,0 +1,39 @@
+const path = require('path')
+const loaders = require('./loaders');
+const plugins = require('./plugins');
+
+module.exports = {
+  entry: path.resolve(__dirname, '../src', 'index.js'),
+  output: {
+    path: path.resolve(__dirname, '../dist'),
+    filename: 'bundle.js'
```

Comment:
You might want to take advantage of substitutions for filenames, to take advantage of [caching](https://webpack.js.org/guides/caching/#output-filenames) for your production builds.